### PR TITLE
basicUI

### PR DESCRIPTION
**Pull request by carlos-ssh**
- Add eslinrc file.
- Add linters setup file for linters in the pull request.
- Add css linter setup file.
- Update README file.
- Inicialized npm and generate a package.json file.
- Add webpack configuration file.
- Add src directory.
- Add basic structure.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      <div id="tablediv" class="tablediv">
+        <!-- Table -->
+        <ul>
+          <li class="li1"><strong> 'Продукты' </strong>, <strong>score:</strong> 100</li>
+          <li class="li2"><strong> 'John Banana' </strong>, <strong>score:</strong> 20</li>
+          <li class="li1"><strong> 'Patrick' </strong>, <strong>score:</strong> 10</li>
+          <li class="li2"><strong> 'Carlos' </strong>, <strong>score:</strong> 50</li>
+          <li class="li1"><strong> 'Oscar' </strong>, <strong>score:</strong> 60</li>
+          <li class="li2"><strong> 'Albert Pike' </strong>, <strong>score:</strong> 40</li>
+          <li class="li1"><strong> 'Robert' </strong>, <strong>score:</strong> 70</li>
```
Comment (eliasecasta) [inline src/index.html]: - [ ] This content has to be rendered with Javascript using the DOM manipulation, please do so to comply with the project requirements. Don't forget to use ES6 syntax.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+body {
```
Comment (eliasecasta) [inline src/style.css:1]: - [ ] Please match the [wireframe ](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png)positioning provided by Microverse to comply with the project requirements.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @carlos-ssh, I am Elias, your reviewer for this PR

Well done so far with your project!

Almost everything is correctly set up like readme and your code overall, keep it up!

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [ ] Please check the comments below in this PR.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @carlos-ssh, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**